### ISSUE TITLE
unsupported ARIA attribute in pagination

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6839]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), pagination includes an ARIA attribute which is not supported. This occurs whenever pagination appears:

- [Search results](https://dev.nurax.samvera.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q=) (all views)
- [User list](https://dev.nurax.samvera.org/users?locale=en) (need admin access to view)
- [My Collections](https://dev.nurax.samvera.org/dashboard/my/collections?locale=en) and [Managed/All Collections](https://dev.nurax.samvera.org/dashboard/collections?locale=en)
- [My Works](https://dev.nurax.samvera.org/dashboard/my/works?locale=en) and [Managed/All Works](https://dev.nurax.samvera.org/dashboard/works?locale=en)
- Review submissions, [under review](https://dev.nurax.samvera.org/admin/workflows?locale=en) and [published](https://dev.nurax.samvera.org/admin/workflows?locale=en&state=published)

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q= and run the Site Improve browser extension
![aria-pagination](https://github.com/samvera/hyrax/assets/32644254/7e7727b5-2b2a-4fed-b19e-4317e62827ce)

**Comment by aahurford** [comment]
I could reproduce in pg but not dev for both search results and all works.

Search results:

![Image](https://github.com/user-attachments/assets/1205f64a-e000-49c0-9f0c-e56b8d0d831a)

![Image](https://github.com/user-attachments/assets/db14435f-7091-42da-8421-1efa3b32f298)

All works: 
![Image](https://github.com/user-attachments/assets/85193c0c-1153-47b5-a380-baef53c84e07)

![Image](https://github.com/user-attachments/assets/1af856d3-6f59-4517-a8a1-9d8fe085d4ca)

**Comment by rodyoukai** [comment]
Hi @rjkati 

Regaring the /catalog view the ARIA attribute is set by Blacklight gem

![image](https://github.com/user-attachments/assets/7b281973-75a3-425e-ab31-e8def7bcee3d)

To address this issue in the Hyrax gem, the only solution would be to override the Blacklight gem's view. Would this approach be acceptable?

**Comment by rkuehn-uofl** [comment]
Unable to reproduce (test links below).

**Hyrax v5.1.0-beta1 Links:**
Example link from above without search term (error - page doesn't exist) -  

1. https://dev.nurax.samvera.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q

Example link from above with search term - 

1. https://dev.nurax.samvera.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q=test

**Koppie v5.1.0-beta1 Links:**

- https://f6.nurax.samvera.org/catalog?search_field=all_fields&q=

- https://f6.nurax.samvera.org/catalog?search_field=all_fields&q=test

<img width="512" alt="Image" src="https://github.com/user-attachments/assets/6f9282b8-6f24-4ee9-bfee-32e8b69d093e" />
